### PR TITLE
.travis.yml: Move build steps into script

### PR DESCRIPTION
**Pull request by jayvdb**
Allow Netlify to execute the same build process.

Closes https://github.com/coala/community/issues/36

### CODE REVIEW

**Comment by jayvdb** [comment]
Note this also means that builds on forks and PRs will fail until the scripts know how to fail gracefully

**Comment by jayvdb** [comment]
The site comes up in Netlify with the following build command

> echo -n coala > org_name.txt && .ci/build.sh

(and publishing `public`)

File: gci/views.py [hunk 1]
```diff
@@ -66,4 +81,4 @@ def index(request):
     s.append('<script src="static/timeago.js"></script>')
     s.append('<script>loadTimeElements()</script>')
 
-    return HttpResponse('\n'.join(s))
+    return s
```
Comment (blazeu) [inline gci/views.py]: Did it work without `.join` ?

**Comment by yukiisbored** [comment]
ack https://github.com/coala/community/pull/37/commits/66b7505b7ded4bb19f59463feea55253535b2401 https://github.com/coala/community/pull/37/commits/6a835c96d51af5fae80e59ad33b84c4bad2ac822 https://github.com/coala/community/pull/37/commits/286f7220f900facef15a314ec797055be17b5db3 https://github.com/coala/community/pull/37/commits/ae7a546158024abfd6d637162bbb80498aff0067 https://github.com/coala/community/pull/37/commits/3695ce8581b09d77ee52d97e373b7e5b205076af

**Comment by yukiisbored** [comment]
deack https://github.com/coala/community/pull/37/commits/3695ce8581b09d77ee52d97e373b7e5b205076af

**Comment by yukiisbored** [comment]
unack https://github.com/coala/community/commit/3695ce8581b09d77ee52d97e373b7e5b205076af

File: gci/views.py (hunk 1 above)
Comment (jayvdb) [reply gci/views.py]: I dont understand.

Comment (jayvdb) [reply gci/views.py]: `s.append` is a list operation, not a string operation.

Comment (blazeu) [reply gci/views.py]: NVM, meant to say why you didn't do join,
but it's called in `index` apparently.
Sorry.

**Comment by blazeu** [comment]
ack https://github.com/coala/community/pull/37/commits/3695ce8581b09d77ee52d97e373b7e5b205076af

**Pull request merged**